src/mmw/js/src/core/layerPicker.js

```diff
@@ -154,28 +154,40 @@ var LayerPickerGroupView = Marionette.LayoutView.extend({
         this.model.set('polling', true);
         $.when(self.model.get('layersDeferred'), $.ajax({ 'url': pointSrcAPIUrl, 'type': 'GET'}))
             .done(function(observationLayers, pointSourceData) {
-                self.model.set('polling', false);
-                self.model.set('error', null);
-                var observationLayersModel = new Backbone.Collection(_.map(observationLayers, function(leafletLayer, display) {
+                self.model.set({
+                    'polling': false,
+                    'error': null,
+                });
+                var observationLayerObjects =_.map(observationLayers, function(leafletLayer, display) {
                     return {
                         leafletLayer: leafletLayer,
                         display: display,
                         active: false,
                     };
-                }));
-
-                var numberOfPoints = JSON.parse(pointSourceData[0]).features.length;
-                observationLayersModel.add({
-                    leafletLayer: pointSourceLayer.Layer.createLayer(pointSourceData[0], self.leafletMap),
-                    display: 'EPA Permitted Point Sources (' + numberOfPoints + ')',
-                    active: false,
-                });
+                }),
+                    observationLayersCollection = new Backbone.Collection(observationLayerObjects);
```

Comment:
I think one of these lines needs to be indented or un-indented so things line up.